### PR TITLE
Correct Feerate Calc and Tweak Funding Feerate

### PR DESCRIPTION
**Pull request by TheBlueMatt**
See individual commits for details. This effectively addresses ##18.

### CODE REVIEW

File: src/bitcoind_client.rs [hunk 1]
```diff
@@ -111,12 +111,11 @@ impl BitcoindClient {
 						)
 						.await
 						.unwrap();
-					match resp.feerate {
-						Some(fee) => fee,
+					match resp.feerate_sat_per_kw {
+						Some(fee) => std::cmp::max(fee, 253),
```
Comment (jkczyz) [inline src/bitcoind_client.rs:115]: Could you make a constant for `253`? Also rename `fee` to `feerate`?

File: src/bitcoind_client.rs [hunk 2]
```diff
@@ -184,7 +184,16 @@ impl BitcoindClient {
 		let mut rpc = self.bitcoind_rpc_client.lock().await;
 
 		let raw_tx_json = serde_json::json!(raw_tx.0);
-		rpc.call_method("fundrawtransaction", &[raw_tx_json]).await.unwrap()
+		let options = serde_json::json!({
+			"fee_rate": self.get_est_sat_per_1000_weight(ConfirmationTarget::Normal) as f64 / 250.0,
```
Comment (jkczyz) [inline src/bitcoind_client.rs:188]: Maybe write out the unit conversion math (`* 4 / 1000`) for guys like me figuring out the math. 😛

**Review by jkczyz — APPROVED** [review]

File: src/convert.rs [hunk 3]
```diff
@@ -61,8 +61,8 @@ impl TryInto<FeeResponse> for JsonResponse {
 		let errored = !self.0["errors"].is_null();
 		Ok(FeeResponse {
 			errored,
-			feerate: match self.0["feerate"].as_f64() {
-				Some(fee) => Some((fee * 100_000_000.0).round() as u32),
+			feerate_sat_per_kw: match self.0["feerate"].as_f64() {
+				Some(fee) => Some((fee * 100_000_000.0 / 4.0).round() as u32),
```
[the remark this answers is not in the document]
Comment (TheBlueMatt) [reply src/convert.rs]: For once, rustfmt's suggestion here made sense.

**Comment by TheBlueMatt** [comment]
Addressed feedback.

File: src/convert.rs (hunk 3 above)
[the remark this answers is not in the document]
Comment (TheBlueMatt) [reply src/convert.rs]: I take that back, it required extra brackets just to maximize, yet again, the total vertical line count used.....its fine enough, though.

**Review by jkczyz — APPROVED** [review]
LGTM. Please squash fixups.

**Comment by TheBlueMatt** [comment]
Squashed.

**Pull request merged**